Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/DataEditorForm.java

```diff
@@ -172,6 +173,8 @@
      */
     private List<Pair<MediaUnit, IncludedStructuralElement>> selectedMedia;
 
+    private static final String DESKTOP_LINK = "/pages/desktop.jsf";
```

Comment:
We do already have a `referringView` in `DataEditorForm.java`. Perhaps it would be better to redirect there (which is on default also set to `desktop`) instead of statically always leading to `DESKTOP_LINK`?